packages/languages/src/common/window-impl.ts

```diff
@@ -28,16 +28,16 @@ export class WindowImpl implements Window {
             return undefined;
         };
         if (type === MessageType.Error) {
-            return this.messageService.error(message, ...actionTitles).then(mapActionType);
+            return this.messageService.error(message, undefined, ...actionTitles).then(mapActionType);
```

Comment:
we should introduce function overload to keep the less verbose API for most common use cases.

Reply:
Overload method added, so no need to use undefined anymore